README.md

```diff
@@ -35,7 +35,7 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-## Kanban board
+## Kanban board - test
```

Comment:
_note: I couldn't find an open PR on the backend to review.  For your next review request, please open a PR on that repo.  It should contain all the content and point from dev to main.  I'll put my backend comments here._

## Backend:

- [ ] Your backend app has no tests.  Kindly add some in order to ensure your app's integrity.

- [ ]  The backend project readme does not comply with the official [Microverse Readme Template](https://github.com/microverseinc/readme-template).  

Kindly, go [here](https://github.com/microverseinc/readme-template), grab the template and apply it to your backend readme.  The documentation that you have there you can move it to another file and link to it on your readme.  💪